scirpy/_tools/_diversity.py

```diff
@@ -12,11 +13,17 @@ def alpha_diversity(
     groupby: str,
     *,
     target_col: str = "clone_id",
+    method: str = "normalized shannon entropy",
     inplace: bool = True,
     key_added: Union[None, str] = None,
 ) -> pd.DataFrame:
     """Computes the alpha diversity of clonotypes within a group.
 
+    Use a method out of normalized shannon entropy, D50, and `scikit-bio’s alpha diversity metrics 
+    <http://scikit-bio.org/docs/0.4.2/generated/skbio.diversity.alpha.html#module-skbio.diversity.alpha >`,
+    
+
+    Normalized shannon entropy:
```

Comment:
Could you please add a short description of D50 (and ideally, a reference) here?